consensus/src/block_storage/pending_votes.rs

```diff
@@ -147,6 +148,7 @@ impl PendingVotes {
             round,
             is_timeout,
         };
+
```

Comment:
To keep consistent with the style of this file, we can remove this formatting change.

Reply:
Done (the formatting change was accidentally introduced).